src/client/containers/ProfilePage/BoardCardDropdown.component.js

```diff
@@ -22,7 +22,6 @@ export default function DropDownMenuMyBoards({ board }) {
           <button
             type="button"
             onClick={async () => {
-              await setclickedBoardInfo(board);
               history.push('/edit-board');
```

Comment:
You need to supply a board id when you route to the edit board. How else will it know which board to edit?

Reply:
so it should be               history.push('/edit-board/${board.id}'); , right 


> You need to supply a board id when you route to the edit board. How else will it know which board to edit?
so it should be        like this        history.push('/edit-board/${board.id}');  Right?

Reply:
Can't remember the exact route name. Ask the person who created the edit board component